events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -95,43 +111,222 @@ <h3>Questions</h3>
+                    catButton.disabled = true;
+                    databaseButton.disabled = false;
+                    beginButton.disabled = true;
+                    endButton.disabled = false;
+                    newQButton.disabled = false;
+                    showQButton.disabled = false;
+                    answerButton.disabled = true;
+                } 
+                else if (event === "noQuestionPostedCustom") {
+                    databaseButton.value = "Custom Catalog" + "   \u25BC";
+                    databaseButton.disabled = false;
+                    typeButton.disabled = true;
+                    diffButton.disabled = true;
+                    catButton.disabled = true;
+                    beginButton.disabled = true;
+                    endButton.disabled = false;
+                    newQButton.disabled = false;
+                    showQButton.disabled = true;
+                    answerButton.disabled = true;
+                } 
+                else if (event === "nothingEntered") {
+                        document.getElementById("question").innerHTML = "You entered an invalid URL, tab name, or have not properly setup your trivia database for access by this application. Try using the 'Default Catalog' instead.";
+                        document.getElementById('catalog').value = "Default Catalog" + "   \u25BC";
+                        databaseButton.disabled = true;
+                        typeButton.disabled = true;
+                        diffButton.disabled = true;
+                        catButton.disabled = true;
+                        beginButton.disabled = false;
+                        endButton.disabled = true;
+                        newQButton.disabled = true;
+                        showQButton.disabled = true;
+                        answerButton.disabled = true;   
+                } else {
+                    try{
                         triviaData = JSON.parse(event);
-                        document.getElementById("question").innerHTML = triviaData.question;
-                        if (triviaData.type === "multiple") {
-                            document.getElementById("answer").innerHTML =  "<p class=\"correct\">" + 
-                                triviaData.correct_answer + "<p>" + 
-                                triviaData.incorrect_answers[0] + "<p>" + 
-                                triviaData.incorrect_answers[1] + "<p>" +
-                                triviaData.incorrect_answers[2];
-                        } else {
-                            document.getElementById("answer").innerHTML = triviaData.correct_answer;
-                        }
-                    } else {
-                        document.getElementById("question").innerHTML = "Check your trivia database, you may have used up all available questions";
+                        console.log(event);
+                        if(triviaData.application !== "trivia") {
```

Comment:
space after `if` - i notice a bunch of small style errors like this throughout the file. please be mindful of spacing!